server/src/resolvers/mutation.js

```diff
@@ -1,6 +1,10 @@
 const jwt = require('jsonwebtoken');
-const APP_SECRET = process.env.SECRET || "";
+const APP_SECRET = process.env.SECRET || " ";
```

Comment:
why add a space?

Reply:
jwt could not be signed with `""`, so I changed it to `" "`.

following error occurs when the value is `""`:
```
secretOrPrivateKey must have a value, Location: [object Object]
```